core/metrics/src/main/resources/io/smallrye/metrics/mapping.yml

```diff
@@ -0,0 +1,110 @@
+base:
```

Comment:
What's the mapping file for, given it wasn't present before?

Reply:
It's defining the server specific metrics. V2 used mapping.yml file for it and v4 used properties files. I pulled out the class that parses the yaml config into [SmallRye](https://github.com/smallrye/smallrye-metrics/pull/1/files#diff-c04dd7f3444f13aa6ceeeed1140a63aaR45) and made v4 use it.